src/libraries/Microsoft.PowerFx.Interpreter/RecalcEngine.cs

```diff
@@ -381,5 +385,44 @@ public void UpdateSupportedFunctions(SymbolTable s)
         {
             SupportedFunctions = s;
         }
+
+        /// <summary>
+        /// Add a set of user-defined formulas and functions to the engine.
+        /// </summary>
+        /// <param name="script">Script containing user defined functions and/or named formulas.</param>
+        /// <param name="parseCulture">Locale to parse user defined script.</param>
+        /// <param name="symbolTable">Extra SymbolTable if needed.</param>

```

Comment:
When would this be useful?

Reply:
This was part of the original implementation.